Source/Core/DolphinQt/Debugger/BreakpointWidget.cpp

```diff
@@ -250,6 +250,7 @@ void BreakpointWidget::OnDelete()
   PowerPC::memchecks.Remove(address);
   Settings::Instance().blockSignals(false);
 
+  emit BreakpointsChanged();
   Update();
```

Comment:
Thoughts on connecting BreakpointsChanged to Update?

Reply:
If by that you mean moving `emit BreakpointsChanged()` in the `Update` function, it will update each view on `showEvent` even when breakpoints haven't changed since it calls `Update`.

Reply:
No, I meant that emitting BreakpointsChanged would call Update. But after thinking about it more that'd lead to Update being called more than once if we added another change signal so forget about this suggestion :)